pages/user/[username]/repos/[repo]/issues/[issueNumber].js

```diff
@@ -64,31 +73,325 @@ function _Comment({ comments, classes }) {
+
+    const alreadyAssigned = issue.assignees.some(
+      assignee => assignee.user.username === collaborator.username
+    );
+
+    const alreadyInAdd = addAssignees.includes(collaborator.username);
+
+    const alreadyInRemove = removeAssignees.includes(collaborator.username);
+
+    if (alreadyAssigned && !alreadyInRemove) {
+      this.setState({
+        removeAssignees: [...removeAssignees, collaborator.username]
+      });
+    } else if (!alreadyAssigned && !alreadyInAdd) {
+      this.setState({
+        addAssignees: [...addAssignees, collaborator.username]
+      });
+    } else if (!alreadyAssigned && alreadyInAdd) {
+      this.setState({
+        addAssignees: addAssignees.filter(
+          addAssignee => addAssignee !== collaborator.username
+        )
+      });
+    } else if (alreadyAssigned && alreadyInRemove) {
+      this.setState({
+        removeAssignees: removeAssignees.filter(
+          removeAssignee => removeAssignee !== collaborator.username
+        )
+      });
+    }
+  };
+
+  render() {
+    const { classes, issue } = this.props;
+    const {
+      anchorEl,
+      open,
+      collaborators,
+      addAssignees,
+      removeAssignees
+    } = this.state;
+    return (
+      <Fragment>
+        <button
+          type="button"
+          onClick={this.togglePopper}
+          className={classes.assigneeButton}
+        >
+          Assignee <SettingsIcon />
+        </button>
+        <div>
+          {issue.assignees.map(assignee => (
+            <div key={assignee.user.id}>
+              <img
+                src={assignee.user.image}
+                className={classes.assigneeImage}
+              />
+              {assignee.user.username}
+            </div>
+          ))}
+        </div>
+        <Popover open={open} anchorEl={anchorEl} onClose={this.togglePopper}>
+          <Paper className={classes.popper}>
+            <div className={classes.paperTop}>
+              <Typography variant="body2" className={classes.assigneeText}>
+                Assign up to 10 people to this issue
+              </Typography>
+              <TextInput />
+            </div>
+            <div className={classes.paperBottom}>
+              {collaborators.map(collaborator => {
+                const assigned =
+                  (issue.assignees.some(
+                    assignee => assignee.user.username === collaborator.username
+                  ) ||
+                    addAssignees.includes(collaborator.username)) &&
+                  !removeAssignees.includes(collaborator.username);
+                return (
+                  <Button
+                    key={collaborator.username}
+                    className={classes.collaboratorDetails}
+                    onClick={() => this.checkAssignee(collaborator)}
+                  >
+                    {assigned && <CheckIcon />}
```

Comment:
All collaborators should be padded regardless whether check mark is present.

Reply:
done